README.md

```diff
@@ -23,6 +23,18 @@ Clone this locally, then in your local workspace of the `NYTPhotoViewer` repo, r
 
 ## Installation
 
+### Swift Package Manager
+
+NYTPhotoViewer may be installed via SPM, by pointing at this repo's URL and.. (more TK)
```

Comment:
thanks for reviewing, matt, i was hoping you would. i tried to add you as a reviewer but github just laughed at me.

door two: "waiting for release and subsequent version number".

i wonder, could this be released without changeing the version number?  we're not changing the functionality of the widget, merely its packaging. or it that me just be lazy?

Reply:
Also, I tried following this suggestion from your #333, but got build errors.

> Worth noting for those less familiar with Swift Package Manger: Although you can in general build Swift packages from the command line with swift build, in this case you need UIKit available so the easiest way to build this package is by opening the folder containing the project up in Xcode (not an Xcode project, the folder itself) which will load the Swift Package and building from Xcode with an iOS device target will succeed.

Reply:
> Also, I tried following this suggestion from your #333, but got build errors.
You got build errors running through Xcode by opening the project folder or from the command line? I've successfully opened the project folder with this branch checked out and built via Xcode but even using the internet's wisdom I did not get `swift build` to link against UIKit successfully. I did not try very hard because building the package via Xcode works fine and we don't really need to support non-Xcode users for a library aimed at iOS I don't think.

Reply:
> i wonder, could this be released without changeing the version number? we're not changing the functionality of the widget, merely its packaging. or it that me just be lazy?

I would create a new version because you're supporting a whole new package manager which I consider to be a "feature" worth celebrating!

That said, the more pragmatic answer (you might already know) is that you need to either create a new version or re-tag the existing version. When SPM grabs the repo and checks out the version tag requested by someone using this package as a dependency, SPM needs to find a Package.swift file in the repository at that tag's commit. In my projects I try to avoid re-tagging versions.

Reply:
I did 
```
git clone https://github.com/nytimes/NYTPhotoViewer.git OpenViaFolder
```
then I used Xcode to open `~/OpenViaFolder/Examples`

<img width="400" alt="open via folder" src="https://user-images.githubusercontent.com/325572/103931947-1bdff500-50ef-11eb-983a-633f086f6a98.png">

but when i tried to build (either target), I got this sort of thing...
<img width="500" alt="errors" src="https://user-images.githubusercontent.com/325572/103932175-70837000-50ef-11eb-89b8-78a0abdaae98.png">

Reply:
I realized what I did wrong. I forgot to switch targets to an iOS device.